src/types/Feed.ts

```diff
@@ -35,6 +37,8 @@ export type FeedValue = number | HexStringWith0x;
 export interface FeedFetcher {
   name: FetcherName;
   symbol?: string;
+  base?: string;
+  quote?: string;
   params?: unknown;
```

Comment:
I think we do know the params types, all fetchers have it, so let's change `unknown` to defined type `FeedFetcherParams` since we doing this cleanup.

Reply:
For all the multiprocessors that are on the code this `params` has a different form. I would refactor this, but I think it requires more thinking. Do you think we can leave it for another PR?

Reply:
lets do in separate PR in future